src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String information = readFromFile(fromFileName);
+        String report = makeReport(information);
+        writeToFile(toFileName, report);
+    }
+
+    public String readFromFile(String fromFileName) {
+        String[] namesOfArticles = {"supply", "buy", "result"};
```

Comment:
readFromFile method shoud **Read From File**
And that is all. Only read and return file content as String. It can be any file.
We do not need `String[] namesOfArticles = {"supply", "buy", "result"};` for reading text from the file.

Reply:
not fixed